charts/my-bloody-jenkins/templates/ingress.yaml

```diff
@@ -1,7 +1,7 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "my-bloody-jenkins.fullname" . -}}
-{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
-apiVersion: networking.k8s.io/v1beta1
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
```

Comment:
please keep the `networking.k8s.io/v1beta1` and add the `networking.k8s.io/v1` as the first one
so it will work in older clusters